centaur/src/main/resources/standardTestCases/three_step.test

```diff
@@ -1,14 +1,20 @@
-name: threeStepCwl
+name: three_step_cwl
 testFormat: workflowsuccess
 workflowType: CWL
 workflowTypeVersion: v1.0
 backends: [Local]
-backendsMode: "only"
 tags: [localdockertest]
 
 files {
   wdl: three_step/three_step.cwl
   inputs: three_step/inputs.json
+  options: three_step/three_step.options
 }
 
-metadata { }
+metadata {
+  "submittedFiles.workflowType": CWL
+  "submittedFiles.workflowTypeVersion": v1.0
+  "calls.wc.executionStatus": Done
+  "calls.ps.executionStatus": Done
+  "calls.cgrep.executionStatus": Done
```

Comment:
Can we test outputs too? To make sure CWL support isn’t regressing

Reply:
Added a new issue https://github.com/broadinstitute/cromwell/issues/2924 that also describes the current issue with 3step.